css/main.css

```diff
@@ -25,5 +25,19 @@ body {
 }
 
 .dot {
-  stroke: #000;
+  stroke: gray;
```

Comment:
Shall I setup the stroke as black? I think black is usually normal color for stroke.